ccd-definition/CaseEvent/CareSupervision/DEPRECATED.json

```diff
@@ -472,5 +472,75 @@
     "CallBackURLAboutToSubmitEvent": "${CCD_DEF_CASE_SERVICE_BASE_URL}/callback/action-cmo/about-to-submit",
     "CallBackURLSubmittedEvent": "${CCD_DEF_CASE_SERVICE_BASE_URL}/callback/action-cmo/submitted",
     "EndButtonLabel": "Save and continue"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "ID": "COMPLY_LOCAL_AUTHORITY",
+    "Name": "Comply with directions",
+    "Description": "Allows Local Authority user access to comply with their directions as well as ones for all parties",
+    "DisplayOrder": 11,
+    "PreConditionState(s)": "PREPARE_FOR_HEARING;ISSUE_RESOLUTION;FINAL_HEARING",
+    "PostConditionState": "*",
+    "SecurityClassification": "Public",
+    "CallBackURLAboutToStartEvent": "${CCD_DEF_CASE_SERVICE_BASE_URL}/callback/comply-with-directions/about-to-start",
```

Comment:
I would remove the end points as they are no longer required, maybe the endbutton label as well. Ideally all that needs to be left here is the bare minimum definition of the event

Reply:
Sure